api/src/main/java/org/pmiops/workbench/firecloud/FireCloudConfig.java

```diff
@@ -220,10 +219,4 @@ public static ApiClient buildApiClient(WorkbenchConfig workbenchConfig) {
         .setReadTimeout(workbenchConfig.firecloud.timeoutInSeconds, TimeUnit.SECONDS);
     return apiClient;
   }
-
-  @Bean
-  @Lazy
-  public IamCredentialsClient getIamCredentialsClient() throws IOException {
```

Comment:
It is a duplicated bean that Spring2 does not like. 
Feels it make sense to use another one:
https://github.com/all-of-us/workbench/blob/master/api/src/main/java/org/pmiops/workbench/google/GoogleConfig.java